src/test/scala/com/fulcrumgenomics/cmdline/ClpTests.scala

```diff
@@ -61,4 +68,32 @@ class ClpTests extends UnitSpec {
   it should "fail and print usage" in {
     new FgBioMain().makeItSo("SomeProgram --with-args=that-dont-exist".split(' ')) should not be 0
   }
+
+  it should "provide command line information to the tool" in {
+    val tmpPath = makeTempFile("tool_info.", ".tab")
+    new FgBioMain().makeItSo(args=s"--async-io true TestClp -i $tmpPath".split(' ')) shouldBe 0
+    val metrics = Metric.read[FgBioToolInfo](tmpPath)
+    metrics should have length 1
+    val metric = metrics.head
+
+    // args/commandLineWithoutDefaults
+    metric.commandLineWithDefaults should include ("fgbio --async-io true") // argument set _prior_ to the tool name
+    metric.commandLineWithDefaults should not include ("fgbio --compression 5") // default argument _prior_ to the tool name
+    metric.commandLineWithoutDefaults should include ("TestClp")
+    metric.commandLineWithoutDefaults should include (s"-i $tmpPath")
+    metric.commandLineWithoutDefaults should not include ("--print-me :none:") // a default argument
+
+    // commandLineWithDefaults
+    metric.commandLineWithDefaults should include ("fgbio --async-io true") // argument set _prior_ to the tool name
+    metric.commandLineWithDefaults should include (" --compression 5") // default argument _prior_ to the tool namee
+    metric.commandLineWithDefaults should include ("TestClp")
+    metric.commandLineWithDefaults should include (s"--info-path $tmpPath")
+    metric.commandLineWithDefaults should include ("--print-me :none:") // a default argument
+
+    // description
+    metric.description shouldBe "A test class"
+
+    // version
+    //metric.version shouldBe "null" // not set in scalatest
```

Comment:
Is this not possible to test?

Reply:
Exactly, it gets put in the manifest in the JAR, which scalatest doesn't have access to